SU2_CFD/include/fluid/CFluidModel.hpp

```diff
@@ -32,6 +32,7 @@
 
 #include "../../../Common/include/CConfig.hpp"
 #include "../../../Common/include/basic_types/datatype_structure.hpp"
+#include "../../../Common/include/containers/CLookUpTable.hpp"
```

Comment:
Forward declare CLookUpTable instead of including it here.
Before class CFluidModel you just add
`class CLookUpTable`